shacl/register.ttl

```diff
@@ -88,6 +87,19 @@ reg:CreatorShape
         # reg:SchemaSameAsProperty
 .
 
+reg:DateTimeNodeShape
```

Comment:
You now introduce `Node` in the name, which we haven’t been using before. Why? Please try to be consistent in naming.

Reply:
Removed "Node" from name for consistency,